### PR TITLE
Gracefully handle missing root podspec when auto-detecting

### PR DESCRIPTION
**Pull request by k0nserv**
cc @segiddins

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # CocoaPods Core Changelog
 
+## master
```
Comment (segiddins) [inline CHANGELOG.md]: `Master`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+
+##### Enhancments
+
+* Gracefully handle missing root podspecs when auto-detecting.  
+  [Hugo Tunius](https://github.com/k0nserv)
```
Comment (segiddins) [inline CHANGELOG.md]: please link to the issue

File: lib/cocoapods-core/podfile/target_definition.rb [hunk 3]
```diff
@@ -695,7 +695,10 @@ def podspec_path_from_options(options)
           podfile.defined_in_file.dirname + name
         elsif options[:autodetect]
           glob_pattern = podfile.defined_in_file.dirname + '*.podspec{,.json}'
-          Pathname.glob(glob_pattern).first
+          path = Pathname.glob(glob_pattern).first
+          raise Informative, 'Could not locate a *.podspec file in the current directory.' unless path
```
Comment (segiddins) [inline lib/cocoapods-core/podfile/target_definition.rb]: `Could not locate a podspec in the current directory. You can specify the path via ...`

File: spec/fixtures/podfile_without_root_podspec/Podfile [hunk 4]
```diff
@@ -0,0 +1 @@
+podspec
```
Comment (segiddins) [inline spec/fixtures/podfile_without_root_podspec/Podfile]: trailing newline

File: CHANGELOG.md (hunk 2 above)
Comment (k0nserv) [inline CHANGELOG.md]: Even though the issue itself is not in core?

Comment (segiddins) [inline CHANGELOG.md]: yeah, it's still fixing the CocoaPods issue

Comment (k0nserv) [inline CHANGELOG.md]: :+1:

**Comment by segiddins** [comment]
Thanks @k0nserv !

**Pull request merged**